src/main/resources/META-INF/gradle-plugins/com.google.cloud.tools.appengine-flexible.properties

```diff
@@ -14,4 +14,4 @@
 # limitations under the License.
 #
 #
-implementation-class=com.google.cloud.tools.gradle.appengine.flexible.AppEngineFlexiblePlugin
+implementation-class=com.google.cloud.tools.gradle.appengine.appyaml.AppEngineAppYamlPlugin
```

Comment:
Do we need to keep this file, given that we have `appengine-appyaml.properties`?

Reply:
This is just to not break people who have explicitly configured flexible... but it might actually be okay to break them.